shell/components/__tests__/NamespaceFilter.test.ts

```diff
@@ -233,7 +235,7 @@ describe('component: NamespaceFilter', () => {
         },
       ];
 
-      await wrapper.find(`[data-testid="namespaces-dropdown"]`).trigger('click');
+      await wrapper.find(`.icon-chevron-down`).trigger('click');
```

Comment:
Not the end of the world and please mark this comment as done if it is hindering you even a little as we do not have so much time, but was it not possible anymore to use the data value as a selector?

Reply:
I should have left out this change entirely, it was a failed partial effort, I ended up keeping that test skipped. I thought it was blocked by https://github.com/rancher/dashboard/issues/11683. I've looked with fresh eyes today and I was able to resolve the issue ultimately in the test. I've reverted it to use the data-testid selector as well, definitely we should be sticking with those where possible.